mne/viz/utils.py

```diff
@@ -1691,6 +1696,15 @@ def _compute_scalings(scalings, inst):
                                  % (key, value))
             continue
         this_data = data[ch_types[key]]
+        if remove_dc:
+            length = 10  # length of segments (in seconds)
```

Comment:
how do you know it's seconds if you don't use sfreq?

Reply:
Right, this was a leftover from my attempt with `Epochs`. Fixed.